Sources/TokamakStaticHTML/Tokens/BuiltinColors.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  File.swift
+//
```

Comment:
```suggestion
```

Reply:
@carson-katri I think this is the only thing that would be great to fix before merging